homeassistant/const.py

```diff
@@ -139,6 +139,9 @@
 CONF_UNIT_OF_MEASUREMENT = 'unit_of_measurement'
 CONF_UNIT_SYSTEM = 'unit_system'
 CONF_URL = 'url'
+CONF_URL_OFF = 'url_off'
```

Comment:
These should not be added to the main const file, just keep them in the platform.